script.js

```diff
@@ -212,6 +212,7 @@ formemail.addEventListener('submit', (e) => {
   }
 });
```

Comment:
- [ ] Great job saving your user's data to localStorage. However, you need to also get that data whenever the user reloads the page.

## EXAMPLE
```
const email = document.querySelector('.form input[name=\'email\']');
const fullName = document.querySelector('.form input[name=\'fullname\']');
console.log(fullName);
const message = document.querySelector('.form textarea[name=\'message\']');

window.onload = () => {
  const showData = JSON.parse(localStorage.getItem('form'));

  if (showData) {
    fullName.value = showData.name;
    email.value = showData.email;
    message.value = showData.message;
  }
};

```